code/game/objects/structures/mystery_box.dm

```diff
@@ -67,6 +67,32 @@ GLOBAL_LIST_INIT(mystery_box_extended, list(
 	/obj/item/circular_saw,
 ))
 
+GLOBAL_LIST_INIT(mystery_magic, list(
+	/obj/item/gun/magic/wand/arcane_barrage,
+	/obj/item/gun/magic/wand/arcane_barrage/blood,
```

Comment:
These two are supposed to be given by a spell, not as independent items

Reply:
Yeah im aware of this, but they work normally from the box. You just lose them if you drop them, which frankly, doesn't particularly bother me.